### DISCUSSION TITLE
Istoctl vs Helm: Which is Preferred for Installation

### DISCUSSION THREAD

**Discussion by ghost** [istio/istio#46953] [Q&A] [answered]
Hello all, I tried searching open and closed discussions for this but was unable to find a similar discussion.

**What is the preferred way to install istio, istioctl or Helm?** 

If Helm is back and not deprecated (see context below), what are the chances that it becomes deprecated again in the future?

### Context:

> Per the docs, the recommended way to install Istio is using istioctl. The Helm way will be deprecated in the future.

Source: https://discuss.istio.io/t/to-istioctl-or-to-helm/4498

#### Another discussion on Reddit:

> - helm listed as to be deprecated in 1.4: https://istio.io/v1.4/docs/setup/install/helm/
> - helm used to be a first class citizen: https://istio.io/v1.5/docs/setup/install/
> - helm removed in 1.6: https://istio.io/v1.6/docs/setup/install/
> - helm back in 1.8: https://istio.io/v1.8/docs/setup/install/ as an alpha apparently. :-)

https://www.reddit.com/r/istio/comments/m6iwy8/why_is_helm_installation_being_deprecated_for/gr6y52y/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

I know that the FAQ states that istioctl is the recommended way of installing Istio, but I was hoping to get extra confirmation that this will continue to be the recommended way.

> The simplest and most qualified installation and management path with high security. This is the community recommended method for most use cases.

https://istio.io/latest/about/faq/#install-method-selection

**Comment by zirain** [comment, marked as answer]
We cannot deprecate helm, it is widely used in the kubernetes community. Both will be supported in the long run.
For those new to istio, try istioctl.
